debian/stretch/foreman/rules

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-include /usr/share/cdbs/1/rules/debhelper.mk
+# include /usr/share/cdbs/1/rules/debhelper.mk
```

Comment:
why?

Reply:
This helper conflicts with `dh` , as I commented above, it's not a problem to not use it if we use `dh`, `bitcoin` did the same thing https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=808969

Reply:
right, so we don't use cdbs at all anymore, correct? let's nuke it from build-depends then too?